USER_MANUAL.md

```diff
@@ -2763,28 +2763,22 @@ list of escape sequences recognized by the formatter:
 | n               | Insert '\n' followed by the current indentation level     |
 | i               | Increase the current indentation level by 1               |
 | d               | Decrease the current indentation level by 1               |
-| c               | Move to the next color in the list of colors for this
-                    production                                                |
-| r               | Reset color to the default foreground color for the
-                    terminal                                                  |
-| an integer      | Print a terminal or nonterminal from the production.
-                    The integer is treated as a 1-based index into the
-                    terminals and nonterminals of the production.
-
-                    If the offset refers to a terminal, move to the next color
-                    in the list of colors for this production, print the value
-                    of that terminal, then reset the color to the default
-                    foreground color for the terminal.
-
-                    If the offset refers to a regular expression terminal, it
-                    is an error.
-
-                    If the offset refers to a nonterminal, print the unparsed
-                    representation of the corresponding child of the current
-                    term.                                                     |
+| c               | Move to the next color in the list of colors for this production |
+| r               | Reset color to the default foreground color for the terminal (See the next subsection below for more information on how colors work.) |
+| an integer      | Print a terminal or nonterminal from the production (See the note below) |
```

Comment:
please make this a subsection rather than simply an extra paragraph with some bold text in front.

Reply:
Done